src/core/core.scale.js

```diff
@@ -537,6 +539,33 @@ module.exports = Element.extend({
 		return rawValue;
 	},
 
+	/**
+	* @private
+	*/
+	_parseValue: function(raw) {
+		var value = this.getRightValue(raw);
+		var start, end;
+
+		if (helpers.isArray(value)) {
+			start = value[0];
+			end = value[1];
+		} else {
+			start = 0;
+			end = value;
+		}
+
+		return {
+			min: Math.min(start, end),
+			max: Math.max(start, end),
+			start: start,
+			end: end
+		};
+	},
+
+	getScaleLabel: function(rawValue) {
+		var v = this._parseValue(rawValue);
+		return v.min === v.max ? v.min : v.min + ' ; ' + v.max;
```

Comment:
I think we should use `start` and `end` here instead. For directional bars it will matter that we keep them in the correct order